src/signTx.c

```diff
@@ -1855,3 +1870,61 @@ void signTx_handleAPDU(
 	VALIDATE(subhandler != NULL, ERR_INVALID_REQUEST_PARAMETERS);
 	subhandler(p2, wireDataBuffer, wireDataSize);
 }
+
+ins_sign_tx_aux_data_context_t* accessAuxDataContext()
+{
+	switch (ctx->stage)
+	{
```

Comment:
Formatting.

Reply:
It's weird. Our formatting rules push the #if macros to the indentation level of the governed code, but remove the macro indentation that is inside an #if. Well, whatever.